### PR TITLE
Upgrade to JMeter 5.4.1

### PR DESCRIPTION
**Pull request by thiagoarrais**
This PR updates to latest available JMeter (5.4.1). It also upgrades the Java Runtime Environment (to 11).

I also took the liberty to change how the JMeter install location is referenced to make future upgrades easier.

### CODE REVIEW

**Review by lopb — DISMISSED** [review]
seems legit!

**Comment by lopb** [comment]
This pull request will be very useful.

**Comment by diegomarangoni** [comment]
hey @thiagoarrais, thanks for the contribution, we will review it internally.

maybe would be better to add version tags before merging this

wdyt @s-radyuk @s4nji ?

File: docker/jmeter-master/launcher.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ run_jmeter_test() {
   [[ "$USE_WORKERS" == "true" ]] && WORKER_OPTS="-R $(getent ahostsv4 "$WORKER_SVC_NAME" | cut -d ' ' -f 1 | sort -u | paste --serial --delimiters ',')"
   echo "=== Running JMeter load generator ==="
 
-  /opt/apache-jmeter-5.0/bin/jmeter.sh -n -t "$FILE" -l results.csv -e -o /results/ -Jserver.rmi.ssl.disable="$SSL_DISABLED" "$WORKER_OPTS" >>output.log 2>&1 &
+  $JMETER_HOME/bin/jmeter.sh -n -t "$FILE" -l results.csv -e -o /results/ -Jserver.rmi.ssl.disable="$SSL_DISABLED" "$WORKER_OPTS" >>output.log 2>&1 &
```
Comment (diegomarangoni) [inline docker/jmeter-master/launcher.sh:8]:
```suggestion
  "$JMETER_HOME"/bin/jmeter.sh -n -t "$FILE" -l results.csv -e -o /results/ -Jserver.rmi.ssl.disable="$SSL_DISABLED" "$WORKER_OPTS" >>output.log 2>&1 &
```
ShellCheck linting is complaining about unquoted env var

Comment (ldeitos) [inline docker/jmeter-master/launcher.sh:8]: @thiagoarrais, adicione " na variável para passar no lint: "$JMETER_HOME"

**Review by ldeitos — CHANGES_REQUESTED** [review]

**Review by diegomarangoni — COMMENTED** [review]
meanwhile, can you fix the linting, please?

**Review by ldeitos — COMMENTED** [review]
LGTM. I think that MR will be useful to make easier upgrade JMeter Version.

**Review by s4nji — APPROVED** [review]

**Review by s4nji — APPROVED** [review]

**Review by diegomarangoni — APPROVED** [review]

**Review by s4nji — APPROVED** [review]

**Review by s4nji — APPROVED** [review]

**Review by diegomarangoni — APPROVED** [review]

**Comment by diegomarangoni** [comment]
hey @ldeitos, as you request some changes on this PR we need your approval here to be able to merge

**Review by ldeitos — APPROVED** [review]
lgtm

**Comment by s4nji** [comment]
Thank you!

**Pull request merged**